src/pytom_tm/template.py

```diff
@@ -37,8 +37,12 @@ def generate_template_from_map(
         )
 
     if filter_to_resolution is None or filter_to_resolution < (2 * output_spacing):
```

Comment:
```suggestion
    if filter_to_resolution is None:
```
Like this right?

Reply:
yeah, forgot to actually pull int the latest code on my test machine...

Reply:
the test do fail on commit e4482cf, as they should xD

Reply:
haha good